sql/moz-fx-data-shared-prod/adjust_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Adjust Derived
+description: |-
+  Derived tables for Adjust data
+dataset_base_acl: derived
+user_facing: false
+labels: {}
+workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:mozilla-confidential  # TODO: @marlene please make sure this is the correct access group.
```

Comment:
If this is the correct access group please remove the TODO comment here.

Reply:
removed. Thanks